install-requirements.sh

```diff
@@ -2,7 +2,15 @@
 
 set -e
 
+# install dependencies
 for file in `ls functions/*/requirements.txt`
 do
   pip install --upgrade -r $file -t `dirname $file` 
 done
+
+# copy shared tools
+pip install --upgrade -r tools/requirements.txt -t tools
+for dir in `ls -d functions/*/`
+do
+  cp -R tools $dir
```

Comment:
I suggest we totally get away from installing requirements for d43-catalog this way, including the requirements installed by pip, and do what we now do in tx-manager-lambda, which uses the "build" hook in the project.json file. This can then install requirements and copy in file for each function. 

We should also use the "clean" function to remove the requirements by running pip uninstall and remove the tools directory, as sometime we may run `apex deploy` locally and don't want all those copies of the libraries laying around. 

See http://apex.run/#supported-hooks and https://github.com/unfoldingWord-dev/tx-manager-lambda/blob/develop/project.develop.json#L9